### PR TITLE
GameDB: Japanese Games

### PR DESCRIPTION
**Pull request by seta-san**
### Description of Changes
adds and fixes some japanese games

### Rationale behind Changes
5chan requests 

### Suggested Testing Steps
hopefully tests should pass

### CODE REVIEW

**Comment by refractionpcsx2** [comment]
In future you don't need to make a new PR, just make another commit, we can squash it down.

File: bin/resources/GameIndex.yaml [hunk 1]
```diff
@@ -33557,6 +33563,11 @@ SLPM-62589:
   name-sort: SIMPLE2000しりーず Vol.72 THE にんきょう
   name-en: "Simple 2000 Series Vol. 72 - Ninhaza"
   region: "NTSC-J"
+SLPM-62590:
+  name: ジーワンジョッキー3 2005年度版 プレミアムパック
+  name-sort: ジーワンジョッキー 3 2005ねんどばん ぷれみあむぱっく
```
Comment (Tokman5) [inline bin/resources/GameIndex.yaml:33568]: Change `ジーワンジョッキー` into `じーわんじょっきー`  plz

**Comment by RedDevilus** [comment]
Prefix is GameDB:

File: bin/resources/GameIndex.yaml [hunk 2]
```diff
@@ -29653,7 +29657,9 @@ SLPM-55244:
     halfPixelOffset: 2 # Fixes blurriness.
     getSkipCount: "GSC_NFSUndercover"
 SLPM-55245:
-  name: "Himawari - Pebble in the Sky"
+  name: ひまわり - Pebble in the Sky -
+  name-sort: ひまわり - Pebble in the Sky -
+  name-en: "Himawari - Pebble in the Sky"
```
Comment (TellowKrinkle) [inline bin/resources/GameIndex.yaml:29662]: No spaces around the dashes for the Japanese title: http://never-island.com/himawari/product.html

**Pull request merged**